setup.py

```diff
@@ -13,10 +14,12 @@
     license='BSD 3-clause',
     url='http://github.com/tensorlib/tensorlib/',
     install_requires=['numpy',
-                      'scipy',],
+                      'scipy', ],
```

Comment:
ah! I didn't know PEP8 said that - what is the rule? Space after comma?

Then I'd just remove the comma. But otherwise LGTM and thanks for proposing this change - it's only a warning in my numpy, but they are very clear about this becoming an error soon!